.github/workflows/build-firmware.yaml

```
@@ -318,10 +318,10 @@ jobs:
       if: ${{ env.skip != 'true' }}
       env:
         ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
-      run: | 
-        wget 'https://developer.arm.com/-/media/Files/downloads/gnu/11.2-2022.02/binrel/gcc-arm-11.2-2022.02-x86_64-arm-none-eabi.tar.xz' -O compiler.tar.xz
+      run: |
+        wget 'https://developer.arm.com/-/media/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi.tar.xz' -O compiler.tar.xz
```

`--progress=dot:giga` will reduce the log size:

```suggestion
        wget 'https://developer.arm.com/-/media/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi.tar.xz' --progress=dot:giga -O compiler.tar.xz
```

https://www.gnu.org/software/wget/manual/html_node/Download-Options.html#Download-Options